.github/workflows/cpu-test.yml

```diff
@@ -0,0 +1,34 @@
+name: cpu-test
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the habana_main branch
+  push:
+    branches:
+      - habana_main
+  pull_request:
+    branches:
+      - habana_main
```

Comment:
what do you think about adding also habana_next? Just temporary until the time we maintain two branches